crates/egui/src/widgets/checkbox.rs

```diff
@@ -38,7 +38,7 @@ impl<'a> Checkbox<'a> {
 
     /// Display an indeterminate state (neither checked nor unchecked)
     ///
-    /// This only affects the checkbox's appearance. It will still toggle its boolean value when
+    /// This only affects the checkboxes appearance. It will still toggle its boolean value when
```

Comment:
This one isn't a typo.

Reply:
Huh, the grammar check marked it as wrong but I guess the original version was correct. I'll revert it